common/spec/domain/sendText.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { SendTextData } from "jslib-common/models/data/sendTextData";
+import { SendText } from "jslib-common/models/domain/sendText";
+
+import { mockEnc } from "../utils";
+
+describe("SendText", () => {
+  let data: SendTextData;
+
+  beforeEach(() => {
+    data = {
+      text: "encText",
+      hidden: false,
+    };
+  });
+
+  it("Convert from empty", () => {
+    const data = new SendTextData();
+    const secureNote = new SendText(data);
+
+    expect(secureNote).toEqual({
+      hidden: undefined,
+      text: null,
+    });
+  });
+
+  it("Convert", () => {
+    const secureNote = new SendText(data);
+
+    expect(secureNote).toEqual({
+      hidden: false,
+      text: { encryptedString: "encText", encryptionType: 0 },
+    });
+  });
+
+  it("Decrypt", async () => {
+    const secureNote = new SendText();
+    (secureNote.text = mockEnc("text")), (secureNote.hidden = true);
```

Comment:
Is there a reason you assigned the properties this way instead of "normally".

Reply:
I accidently used `,` instead of `;` which prettier formated like this. 😞